druid-shell/src/backend/windows/clipboard.rs

```diff
@@ -18,20 +18,26 @@ use std::ffi::CString;
 use std::mem;
 use std::ptr;
 
+use winapi::shared::minwindef::MAX_PATH;
 use winapi::shared::minwindef::{FALSE, UINT};
```

Comment:
Let's group these imports.

```suggestion
use winapi::shared::minwindef::{MAX_PATH, FALSE, UINT};
```